src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -88,18 +91,30 @@ const Packages = ({ defaultArch, ...props }) => {
         });
     });
 
-    // call api to list available packages
-    const handlePackagesAvailableSearch = async () => {
-        const { data } = await api.getPackages(
+    const getAllPackages = async () => {
+        const args = [
             getState()?.values?.release,
             getState()?.values?.architecture || defaultArch,
             packagesSearchName.current
-        );
-        if (data) {
-            const packagesAvailableFiltered = filterPackagesAvailable(data);
+        ];
+        let { data, meta } = await api.getPackages(...args);
+        if (data && meta.count === data.length) {
```

Comment:
Thanks to [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) you don't need to check if data exists. You can just do `data?.length`